### PR TITLE
remove the definition of the image/avif-sequence MIME type

### LINKED ISSUE #59: Potential issues with two MIME types
1. There's no precedent for image formats with distinct MIME types for their still and animated variants, e.g., GIF, PNG, and WebP(?). Why was a different decision made here?
2. As far as I could tell from the document there's no distinct processing model defined for consumers. Producers are told to use specific MIME types, but there are no requirements for consumers getting MIME types with certain bits being wrong. Absence such requirements, consumers will likely use the same decoding path for both, and producers might eventually stop caring about the difference as well as there's no discernible difference.

### PR DESCRIPTION
**Pull request by cconcolato**
Closes #59

***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/av1-avif/pull/86.html" title="Last updated on Aug 18, 2020, 2:00 PM UTC (1a01ef2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/av1-avif/86/64b4a87...1a01ef2.html" title="Last updated on Aug 18, 2020, 2:00 PM UTC (1a01ef2)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -376,7 +331,7 @@ Fragment identifiers are specified in Annex L of ISO/IEC 14496-12, available as
 
 <strong>Additional information:</strong>
     - <strong>Magic number(s):</strong> none
-    - <strong>File extension(s):</strong> avifs, heifs or hif
+    - <strong>File extension(s):</strong> avif, avifs, heif, heifs or hif
```
Comment (aklemets) [inline index.bs:334]: Remove avifs from list here

**Comment by cconcolato** [comment]
We probably need to do a version change like v1.0.1 and change the date. Any view on that? I'll do that as a final step before merging.

**Comment by aklemets** [comment]
Updating the spec to v1.0.1 sounds fine to me.

**Comment by negge** [comment]
Depending on how significant any normative changes are, we may want to consider using the same updated version as the AV1 spec:

Version 1.0.0 with Errata 1

**Pull request merged**